lib/settings.coffee

```diff
@@ -1,26 +1,42 @@
+
```

Comment:
`main.coffee`? I guess most packages use the package name as the main file, but I thought to avoid that so it's consistent with the light theme.